netlify.toml

```diff
@@ -1032,3 +1032,105 @@
 [[redirects]]
     from = "/handbook/company/branding"
     to = "/handbook/company/brand-assets"
+
```

Comment:
@joethreepwood are these redirects supposed to be part of this PR?

Reply:
I haven't added them in, no. They seem to have been added in by the safe redirects bot and I have no idea why?